src/dialogs/examples/basic/O365ConnectorCardActionsDialogV1.ts

```diff
@@ -0,0 +1,206 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as teams from "botbuilder-teams";
+
+export class O365ConnectorCardActionsDialogV1 extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        // multiple choice examples
```

Comment:
Yeah, this needs more documentation. Examples of what? Add comment to the input saying what it produces, e,g,
// compact, multiselect, required